### PR TITLE
Made the portfolio webpage Responsive for desktop

### PR DESCRIPTION
**Pull request by HossainAraf**
- Used media queries in the CSS file to make the  desktop version look like the figma template (Desktop: >=768px);
- In order to lay out the elements  Grid and Flexbox are implemented;
- Buttons are made interactive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -202,7 +191,7 @@ <h2>About Myself</h2>
     </section>
     <!-- Form -->
     <section id="form" class="flex">
-      <form class="flex-column" action="https://formspree.io/f/mrgvnqgw" method="post">
+      <form class="flex-column form" action="https://formspree.io/f/mrgvnqgw" method="post">
```
Comment (Kossi-stack) [inline index.html:194]: - [ ] Kindly make sure the background image covers the screen
- [ ] Your Design
Screenshot 2023-06-15 at 04 28 10

- [ ] Figma design
Screenshot 2023-06-15 at 03 48 54

File: index.html [hunk 2]
```diff
@@ -67,17 +67,6 @@ <h4>CANOPY</h4>
...
-        <div class="work-description flex">
-          <ul class="flex">
-            <li class="language-tag flex"><p>HTML</p></li>
-            <li class="language-tag flex"><p>CSS</p></li>
-            <li class="language-tag flex"><p>Javascript</p></li>
-          </ul>
-        </div>
-        <button class="see-project-btn">See Project</button>
       </div>
       <div class="work-first">
```
Comment (Kossi-stack) [inline index.html:71]: - [ ] Kindly make sure the content and the image are on the same level. You need to reduce margin-top for Tonic 
- [ ] Your design 
Screenshot 2023-06-15 at 04 14 43

Figma Design
Screenshot 2023-06-15 at 04 42 10

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @HossainAraf,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

- [ ] Kindly make sure to follow the design
- [ ] Your Design
Screenshot 2023-06-15 at 04 13 08

- [ ] Figma Design
Screenshot 2023-06-15 at 04 54 07

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @HossainAraf,

I am Favour 👩🏽 , your reviewer for this project milestone.

Great Job implementing the required changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- Good commit messages ✔️
- Professional Readme file ✔️
- Portfolio is responsive ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**